packages/web/src/scss/components/Accordion/index.html

```diff
@@ -106,6 +106,7 @@ <h2 class="docs-Heading">Default accordion (Always open)</h2>
           id="accordionExample_article_1_collapse"
           class="Collapse is-open"
           aria-labelledby="accordionExample_article_1_header"
+          style="height: 100%"
```

Comment:
can't we set this to the `.Collapse.is-open:not(.is-transitioning)` class directly? I don't like it here tbh.

Reply:
That height is there purely as a precaution so that it flashes when it is supposed to be open.